src/index.html

```diff
@@ -8,7 +8,205 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <div id="header" class="header-wrapper">
+        <ul id="main-nav">
+          <li class="main-nav-item"><a href="">Home</a></li>
+        </ul>
```

Comment:
For the page size and rendering performance, we need to keep the tags inheritance hierarchy as simple as possible. So the list wrapper here is a kind of redundant.

Besides, instead of using relative path, it is better to use absolute `/` for navigating to the homepage.

```html
<header id="header">
  <a href="/">Home</a>
  <button>Download my full resume</button>
</header>
```